packages/wallet-specs/src/protocols/virtual-defunding-as-hub/protocol.ts

```diff
@@ -17,23 +13,24 @@ type ChannelsSet = Init & {
   ledgerChannelIds: IDs;
 };
 
-export const assignChannels = assign(
-  ({ jointChannelId }: Init): ChannelsSet => {
-    const { guarantorChannelIds } = checkThat(store.getEntry(jointChannelId).funding, isGuarantee);
+// TODO: Use this in machine factory
```

Comment:
Worth creating an issue for this?

Reply:
I think this is fine as a TODO: there are so many TODOs in this package that it would pollute the issues.